### PR TITLE
24.0.1 final changes

### PR DESCRIPTION
**Pull request by fanquake**
Changes for 24.0.1.

At this stage, I don't think an rc is necessary here, and would just add another 5-x days delay to fixing these issues.

This will be accompanied by a pull request to master to add a note in the 24.0.md that already exists there. Similar to what was done with 0.19.0.1.

### CODE REVIEW

**Review by hebasto — APPROVED** [review]
ACK a9ea7158351ad89f62a13335368df8cc54114f7f, I have reviewed the changes and they look OK.

File: doc/man/bitcoin-cli.1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.2.
```
Comment (josibake) [inline doc/man/bitcoin-cli.1:1]: not super familiar with this process, but im guessing modifying these files won't cause any issues?

**Review by josibake — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/26647/commits/a9ea7158351ad89f62a13335368df8cc54114f7f

lgtm, altho im assuming its okay to modify `doc/man` files because they are generated one time as part of creating the `24.x` branch

**Comment by gruve-p** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/26647/commits/a9ea7158351ad89f62a13335368df8cc54114f7f

**Comment by instagibbs** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/26647/commits/a9ea7158351ad89f62a13335368df8cc54114f7f

**Review by w0xlt — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/26647/commits/a9ea7158351ad89f62a13335368df8cc54114f7f

**Comment by Collinschimaobi** [comment]
0bf3013

**Pull request merged**